### ISSUE TITLE
error when click button ask afip

### ISSUE THREAD

**Issue by lukio** [tryton-ar/recover_invoice_ar#4]
```python
File "/pysimplesoap/client.py", line 469, in wsdl_parse
    xml = fetch(url)
  File "/pysimplesoap/client.py", line 465, in fetch
    warnings.warn("Cannot write file", filename)
TypeError: issubclass() arg 1 must be a class
```
![recover_invoice_error_pysimplesoap](https://user-images.githubusercontent.com/776682/34530553-44f1508e-f08e-11e7-86db-2a7e02fa0ee1.png)

**Comment by lukio** [comment]
Set the cache dir when called to Conectar method from pyafipws.
The problem was that pyafipws/cache dir it's not writable from the user that executes the trytond daemon.

**Comment by lukio** [comment]
Fixes with commit: 7be450abff63ed5273c001cd74a165238c9d5c7c
> set cache dir from account_invoice_ar when Connect to WSDL afip
